### PR TITLE
fix(gatsby-remark-images-contentful): fix image urls with https/http prefix

### PR DESCRIPTION
**Pull request by markza**
This pull request fixes scenario where Contentful markdown which includes images with urls with `https:` prefix were leading to forbidden 403 errors on Gatsby startup.

Urls with `http:` prefix will also now render correctly.

The default in Contentful is to insert images without `http/https` prefix.

### CODE REVIEW

File: packages/gatsby-remark-images-contentful/src/index.js [hunk 1]
```diff
@@ -91,7 +96,7 @@ module.exports = async (
   <span
     class="gatsby-resp-image-wrapper"
     style="position: relative; display: block; ${
-      options.wrapperStyle
+      options.showCaptions ? null : options.wrapperStyle
```
Comment (pieh) [inline packages/gatsby-remark-images-contentful/src/index.js]: should this be here? it seems unrelated

Comment (markza) [reply packages/gatsby-remark-images-contentful/src/index.js]: Yes, it's unrelated to the bug, I forgot I added it with this PR, sorry about that.
I noticed this was handled differently in `gatsby-remark-images` and brought it in line with that code.

**Comment by m-allanson** [comment]
I've fixed the merge conflicts from `master` and removed that one unrelated change. This is good for another review @gatsbyjs/core 👍

File: packages/gatsby-remark-images-contentful/src/index.js (hunk 1 above)
Comment (m-allanson) [reply packages/gatsby-remark-images-contentful/src/index.js]: I've removed this change, let's add it in a separate PR.
